p2p/controller_connections.go

```diff
@@ -301,7 +301,7 @@ func (c *controller) Dial(ep Endpoint) (*Peer, []Endpoint) {
 	c.logger.Debugf("Dialing to %s", ep)
 	con, err := c.dialer.Dial(ep)
 	if err != nil {
-		c.logger.WithError(err).Infof("Failed to dial to %s", ep)
+		c.logger.WithError(err).Warnf("Failed to dial to %s", ep)
```

Comment:
Not being able to connect to a node is something that is normal, such as when a node is behind NAT/firewall and can't be dialed to. It doesn't necessarily indicate something is going wrong. I was going to say it should be moved to Debug but after writing out this reply, I think a "warning" makes sense in the context. Nothing's wrong but if it happens constantly something might be.

It might get spammy, however.

Reply:
That was my intention with that Warning, it's not a problem but you should pay attention, maybe blacklist a node that you constantly fail to connect to.